configuration/ibm/50001001.json

```diff
@@ -876,24 +876,29 @@
         "/sys/bus/i2c/drivers/at24/7-0051/eeprom": [
             {
                 "inventoryPath": "/system/chassis/motherboard/lcd_op_panel_hill",
-                "devAddress": "7-0051",
-                "driverType": "at24",
-                "busType": "i2c",
-                "presence": {
-                    "pollingRequired": true,
-                    "pin": "RUSSEL_OPPANEL_PRESENCE_N",
-                    "value": 0,
-                    "gpioI2CAddress": "0-0020"
-                },
+                "gpioI2CAddress": "0-0020",
                 "preAction": {
-                    "pin": "RUSSEL_FW_I2C_ENABLE_N",
-                    "value": 0,
-                    "gpioI2CAddress": "0-0020"
+                    "collection": {
+                        "gpioPresence": {
+                            "pin": "RUSSEL_OPPANEL_PRESENCE_N",
+                            "value": 0
+                        },
+                        "setGpio": {
+                            "pin": "RUSSEL_FW_I2C_ENABLE_N",
+                            "value": 0
+                        },
+                        "systemCmd": {
+                            "cmd": "echo 7-0051 > /sys/bus/i2c/drivers/at24/bind"
+                        }
+                    }
                 },
                 "postActionFail": {
-                    "pin": "RUSSEL_FW_I2C_ENABLE_N",
-                    "value": 1,
-                    "gpioI2CAddress": "0-0020"
+                    "collection": {
```

Comment:
why is the indentation mixed here ?

Reply:
It is not mixed, It is correct. Its just that the preAction length is small than postActionFail so it looks like mixed in comparison to that. :)